cypress/e2e/clue/branch/student_tests/dataflow_tool_spec.js

```diff
@@ -386,16 +386,35 @@ context('Dataflow Tool Tile', function () {
           dataflowToolTile.getDropdownOptions(nodeType, dropdown).eq(0).click();
       });
       it("can connect and trigger modal connection warning", () => {
+        // vars for controlling click and drag to connect nodes
+        const startX = 306;
```

Comment:
I had to adjust these for the current state of Dataflow node styles, so I made it configurable.